CHANGELOG.md

```diff
@@ -1,12 +1,9 @@
-## 2.2.0-nullsafety-dev
+## 3.0.0-nullsafety-dev
 
 Pre-release for the null safety migration of this package.
 
-Note that 2.2.0 may not be the final stable null safety release version, we
-reserve the right to release it as a 3.0.0 breaking change.
-
 This release will be pinned to only allow pre-release sdk versions starting from
-2.10.0-2.0.dev, which is the first version where this package will appear in the
+2.12.0-0, which is the first version where this package will appear in the
 null safety allow list.
```

Comment:
This entire paragraph can be removed. We are no longer pinning to pre-release, and the allow list isn't a factor in the `2.12.0` SDKs.

Reply:
ack